src/views/utils.js

```diff
@@ -65,36 +65,40 @@ export function getImageLayers(id, props) {
     ...layerProps
   } = props;
   const { loader } = layerProps;
+
   // Create at least one layer even without loaderSelection so that the tests pass.
-  if (loader.isPyramid) {
-    return [loaderSelection, newLoaderSelection]
-      .filter((s, i) => i === 0 || s)
-      .map((s, i) => {
-        const suffix = s
-          ? `-${transitionFields.map(f => s[0][f]).join('-')}`
-          : '';
-        const newProps =
-          i !== 0
-            ? {
-                onViewportLoad,
-                refinementStrategy: 'never',
-                excludeBackground: true
-              }
-            : {};
-        return new MultiscaleImageLayer({
-          ...layerProps,
-          ...newProps,
-          loaderSelection: s,
-          id: `${loader.type}${getVivId(id)}${suffix}`,
-          viewportId: id
-        });
-      });
+  if (loader.length === 1) {
+    return [
+      new ImageLayer(layerProps, {
+        id: `${loader.type}${getVivId(id)}`,
+        viewportId: id,
+        loaderSelection,
+        loader: loader[0] // should just be a pixel source
+      })
+    ];
   }
-  return [
-    new ImageLayer(layerProps, {
-      id: `${loader.type}${getVivId(id)}`,
-      viewportId: id,
-      loaderSelection
-    })
-  ];
+  // isPyramid
+  return [loaderSelection, newLoaderSelection]
+    .filter((s, i) => i === 0 || s)
+    .map((s, i) => {
+      const suffix = s
+        ? `-${transitionFields.map(f => s[0][f]).join('-')}`
+        : '';
+      const newProps =
+        i !== 0
+          ? {
+              onViewportLoad,
+              refinementStrategy: 'never',
+              excludeBackground: true
+            }
+          : {};
+      return new MultiscaleImageLayer({
+        ...layerProps,
+        ...newProps,
+        loaderSelection: s,
+        id: `${loader.type}${getVivId(id)}${suffix}`,
```

Comment:
Same: Need to change this `loader.type` since it doesn't currently exist.  Might be worth adding this to the interface, even optionally, though.